lib/KCard/index.vue

```diff
@@ -0,0 +1,393 @@
+<template>
+
+  <BaseCard
+    :to="to"
+    :title="title"
+    :headingLevel="headingLevel"
+    :titleLines="titleLines"
+    :class="['k-card', rootClass]"
+    :headingStyles="headingStyles"
+  >
+    <template v-if="$slots.title" #title>
+      <!-- @slot Optional slot section containing the title contents, should not contain a heading element. -->
+      <slot name="title"></slot>
+    </template>
+    <template #default>
+      <div class="thumbnail">
+        <!-- 
+          Render KImg even if thumbnailSrc is not provided since in that case
+          KImg takes care of showing the gray placeholder area.
+        -->
+        <KImg
+          :src="thumbnailSrc"
+          :scaleType="thumbnailScaleType"
+          :aspectRatio="thumbnailAspectRatio"
+          :isDecorative="true"
+          :appearanceOverrides="thumbnailStyles"
+        />
+        <!--
+          This is a duplicate of the same slot in KImg. I didn't find a way to utilize
+          KImg's placeholder slot from here, likely because this part of code is nested
+          in one slot already
+
+          Show it even when thumbnail source is provided - then the placeholder serves
+          as progressive loading experience.
+        -->
+        <span
+          v-if="$slots.thumbnailPlaceholder"
+          class="thumbnail-placeholder"
+        >
+          <!-- @slot Places content to the thumbnail placeholder area. -->
+          <slot name="thumbnailPlaceholder"></slot>
+        </span>
+      </div>
+      <div
+        data-test="aboveTitle"
+        class="above-title"
+      >
+        <slot name="aboveTitle"></slot>
```

Comment:
I think we should document these slots as well.

Reply:
Ah yes, thank you. Done